packages/app/src/specs/InlineSpecList.vue

```diff
@@ -14,9 +14,9 @@
         />
       </template>
       <template v-else>
-        <div class="text-white">
-          FileTree not implemented
-        </div>
+        <InlineSpecListTree
+          :specs="specs.map(spec => spec.node)"
+        />
       </template>
```

Comment:
Why use a `<template>` here if there is only going to be one element?
Same above.

Reply:
Fixed!